### ISSUE TITLE
JSON import of circuit with terminations fails without error

### ISSUE THREAD

**Issue by candlerb** [netbox-community/netbox#12402]
### NetBox version

v3.4.10

### Python version

3.8

### Steps to Reproduce

1. Create circuit type and provider (I am using "Internet" and "Cerberus" in the example below)
2. Go to Circuits > Import
3. Paste in the following:

```
[
    {
        "cid": "ID-TEST",
        "commit_rate": 200000,
        "description": "Testing Import",
        "provider": "Cerberus",
        "status": "decommissioned",
        "terminations": [
            {
                "site": {
                    "name": "Test"
                }
            }
        ],
        "type": "Internet"
    }
]
```

### Expected Behavior

*Either* the import to complete successfully, *or* an error message to be displayed.

### Observed Behavior

Netbox returns to the import page, without showing any error, and without completing the import.

Aside 1: if I give a provider name or circuit type that doesn't exist, this *is* correctly reported as an error.

Aside 2: I can find no documented example of the correct format for this, so I'm just going by guesswork - but if my guess is wrong, I think Netbox should at least say what's wrong.

Related issues: #11469, #11204, #815.

**Comment by candlerb** [comment]
If I set `"terminations": []" or omit the 'terminations' member entirely, then import succeeds. So this does appear to be something to do with processing the terminations.

**Comment by candlerb** [comment]
I was able to find a structure which works:

```
[

{
"cid": "ID-TEST",
"provider": "Cerberus",
"type": "Internet",
"commit_rate": 200000,
"status":"decommissioned",
"description": "Testing Import",
"terminations": [{
"site": "Test",
"term_side": "A"
}]
}

]
```

Although the import confirmation screen shows no side A or side Z:

![image](https://user-images.githubusercontent.com/44789/235372302-49833adc-501a-4699-bc92-cdcdc948ab56.png)

... in fact the circuit *and* termination were created correctly (so this is mainly a cosmetic issue).

The main issue still stands though: if I change the import so the termination is invalid, for example by changing the "site" to a non-existent name, there is no error feedback.

**Comment by arthanson** [comment]
@candlerb This looks like it has been fixed, I tried with NetBox v3.7 and it correctly throws an error popup both about the format and the site not existing.  Closing now as it looks fixed, but please let me know if you still have an issue.
